src/material/core/focus-indicators/_focus-indicators.scss

```diff
@@ -66,24 +78,24 @@ $mat-focus-indicator-border-style: solid;
 
 /// Mixin that sets the color of the focus indicators.
 ///
-/// @param {color|map} $themeOrMap
+/// @param {color|map} $theme-or-color
 ///   If theme, focus indicators are set to the primary color of the theme. If
 ///   color, focus indicators are set to that color.
 ///
 /// @example
 ///   .demo-dark-theme {
-///     @include mat-strong-focus-indicators-theme($darkThemeMap);
+///     @include mat-strong-focus-indicators-theme($dark-theme-map);
 ///   }
 ///
 /// @example
 ///   .demo-red-theme {
-///     @include mat-strong-focus-indicators-theme(#F00);
+///     @include mat-strong-focus-indicators-theme(#f00);
 ///   }
-@mixin mat-strong-focus-indicators-theme($themeOrColor) {
+@mixin mat-strong-focus-indicators-theme($theme-or-color) {
```

Comment:
@jelbourn & @mmalerba: Do we still want to have a standalone `mat-strong-focus-indicators-theme` mixin? One option is to merge this into the `mat-strong-focus-indicators` mixin (i.e. add a `border-color` key into the `$config`), but then if developers want to toggle focus indicator styles on light/dark backgrounds, a bunch of redundant styles are re-declared. Thoughts?

Reply:
IMO it makes sense to keep it, because the color depends on the theme.